coderd/database/dbauthz/dbauthz.go

```diff
@@ -1016,6 +1016,12 @@ func (q *querier) GetDERPMeshKey(ctx context.Context) (string, error) {
 	return q.db.GetDERPMeshKey(ctx)
 }
 
+func (q *querier) GetDefaultOrganization(ctx context.Context) (database.Organization, error) {
+	return fetch(q.log, q.auth, func(ctx context.Context, _ any) (database.Organization, error) {
```

Comment:
That does do auth! `fetch` is the helper function that automatically checks the RBAC on the returned object. So if the caller cannot read the default org, this will fail.